Marsiple/Common/Models/Post.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  Post.swift
+//  Marsiple
+//
+//  Created by Matej Korman on 16/07/2018.
+//  Copyright © 2018 Matej Korman. All rights reserved.
+//
+
+import Foundation
+
+class Post {
```

Comment:
The `posts/fetching-data` branch PR contains the full `Post` definition that's missing here. Be careful when merging, to not overwrite the `Codable` implementation.